### DISCUSSION TITLE
how to connect account abstraction wallet using wallet connect. I am building wallet for users but its smart contract wallet.

### DISCUSSION THREAD

**Discussion by aditya-passbird** [WalletConnect/walletconnect-monorepo#3250] [Developer Support] [answered]
I want to know how I can use that, while trying to sign on opensea, it didnt work.

**Comment by arthur-zhuk** [comment]
@aditya-passbird We're you able to figure this out? I'm having the same issue.

**Comment by kikoncuo** [comment]
The problem is with opensea as they still can't verify smart contract signatures

**Comment by glitch-txs** [comment, marked as answer]
We'll get in touch with them

**Comment by glitch-txs** [comment]
Could you guys confirm the exact domain you're having issues with?
